### DISCUSSION TITLE
Feature Request | Support input_boolean for auto order

### DISCUSSION THREAD

**Discussion by fixtse** [Clooos/Bubble-Card#317] [Feature requests]
Hey, I know that you already have enough going on with the project BUT
I wanted to put this out here, so you'll have it in mind.  
Would it be possible to add support for other types of sensor other than the binary_sensor for the Horizontal Stack Auto order.

I was originally thinking of support for something like espresense that provides room presence detection, but that would add a ton of work just to add the options on the UI editor, so instead of that, I was thinking that if we can get support for something like an input_boolean, the user can create automations to control the input_booleand on/off to get an auto order behavior for complex scenarios, without having to load all the logic into the card itself.

Thank you for the cards btw, even though you kept making changes that broke the video I was making several times 😂, the process is much clearer now, and the user has the option to implement it based on their needs and knowledge. A win for everyone.

**Comment by Clooos** [comment]
Sorry for that changes! 🫣 But I'm glad that you like how it is now! 

For your feature request, I've just looked at my code and it should work with input_boolean as well, just change that in YAML.

Keep me informed!

**Comment by Clooos** [comment]
Does it worked?

**Comment by fixtse** [comment]
Hey, yes, just tried it, works perfectly 🎉

**Comment by Clooos** [comment]
Yeah! I will change that in the GUI editor as well.
